css/main.css

```diff
@@ -145,68 +149,77 @@ kbd {
 	display: block;
 }
 
+.flexTopBar {
+	display: flex;
+	flex-direction: column;
+	width: 100%;
+	top: 0;
+	left: 0;
+	z-index: 1;/*Lets topbar render ontop of iframe*/
```

Comment:
same as above, so long as nothing is overlapping we don't need this z-index

Reply:
This one does actually need the z-index to keep the dropdown tabs above the contentContainer. The other was redundant though.